src/store/substrate/index.js

```diff
@@ -135,7 +135,7 @@ export default {
         api.query.system.events((events) => {
           events.forEach((record) => {
             const { event } = record
-            if (event.section == "serviceRequest" || event.section == "rewards" || event.section == "orders" || event.section == "geneticTesting" || event.section == "balances" || event.section == "electronicMedicalRecord" || event.section == "geneticData" || event.section == "geneticAnalysisOrders" || event.section == "geneticAnalysis") {
+            if (event.section == "serviceRequest" || event.section == "rewards" || event.section == "orders" || event.section == "geneticTesting" || event.section == "balances" || event.section == "electronicMedicalRecord" || event.section == "geneticData" || event.section == "geneticAnalysisOrders" || event.section == "geneticAnalysis" || event.section == "geneticAnalysts") {
```

Comment:
Consider refactoring this using a JSON object with `true` values.

```
{ "serviceRequest": true }
```

Here is an example.

Reply:
How about just using an array, I think it's easier to use an array. So we just need to equate its contents with `includes()`. WDYT?

```json
sections = [
  "serviceRequest",
  "rewards",
  ...
]
```
`sections.includes(event.section)`